baybe/utils/dataframe.py

```diff
@@ -418,3 +418,23 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_print_df(df: pd.DataFrame) -> str:
+    """Convert a dataframe into a pretty/readable str object.
+
+    This function returns the first 10 rows and 7 columns from the dataframe.
```

Comment:
This is technically not correct, since the method returns a string containing the rows and columns, and not the rows/columns itself. Also, It does not return the first rows/columns but at most this number of rows/columns.

Reply:
Fixed. Thank you for your suggestion!